Modules/xASL_module_Population.m

```diff
@@ -201,75 +201,84 @@
 %% 7    ROI statistics
 if ~x.mutex.HasState(StateName{8})
     
-    % Default & Fallback for Atlases
+    % Default atlases/ROIs
     if ~isfield(x.S,'Atlases')
         x.S.Atlases = {'TotalGM','DeepWM'}; % Default
     end
-
+    
+    % Default datatypes
+    if ~isfield(x.S,'DataTypes')
+        x.S.DataTypes = {'qCBF'}; % Default
+    end
+    
+    
     x = xASL_init_LoadMetadata(x); % Add statistical variables, if there are new ones
     % if exist('ASL','var')
     %     xASL_vis_OverlapT1_ASL(x, ASL.Data.data); % Overlap T1 GM probability map & CBF, Create image showing spatial/visual agreement between T1 GM segmentation & ASL
     % end
 
     xASL_stat_ComputeWsCV(x); % This computes wsCV & bsCV to compute power   
-    
-    % Iterate over atlases in cell structure
-    
+
+
     % ROI statistics
     % x.S.SubjectWiseVisualization =1; % set this on to visualize the subject-wise masks
     % over CBF maps (takes lot of extra time though)
     
-    % x.S.InputDataStr              = 'SD'; % 'SD' 'TT' 'M0' 'R1' 'ASL_HctCohort' 'ASL_HctCorrInd'
-    % x.S.InputAtlasPath            = fullfile(x.D.MapsSPMmodifiedDir,'WholeBrain.nii');
-    % % x.S.InputAtlasPath            = fullfile(x.D.AtlasDir,'HOcort_CONN.nii');
-    % xASL_wrp_GetROIstatistics( x);
-    % x.S.InputDataStr              = 'SNR'; % 'SD' 'TT' 'M0' 'R1' 'ASL_HctCohort' 'ASL_HctCorrInd'
-    % x.S.InputAtlasPath            = fullfile(x.D.MapsSPMmodifiedDir,'WholeBrain.nii');
-    % xASL_wrp_GetROIstatistics( x);
-
-    x.S.InputDataStr = 'qCBF'; % 'SD' 'TT' 'M0' 'R1' 'ASL_HctCohort' 'ASL_HctCorrInd'
-	x.S.InputDataStrNative = 'CBF'; % 'SD' 'TT' 'M0' 'R1' 'ASL_HctCohort' 'ASL_HctCorrInd'
+    x.S.InputDataStr = 'qCBF'; % 'Texch' 'ATT' 'SD' 'TT' 'M0' 'R1' 'ASL_HctCohort' 'ASL_HctCorrInd'

```

Comment:
These lines can be probably commented out. This value will be always overwritten.

Reply:
Done